tanner/api/api.py

```diff
@@ -77,110 +97,43 @@ async def return_snare_info(self, uuid):
 
             query_res = []
             async with self.pg_client.acquire() as conn:
-                async with conn.cursor() as cur:
-                    await cur.execute(
-                        """
-                    SELECT * FROM sessions
-                    WHERE
-                        sessions.sensor_id = '%s'
-                    """
-                        % (uuid)
+                stmt = select([SESSIONS]).where(SESSIONS.c.sensor_id == uuid)
+                query = await (await conn.execute(stmt)).fetchall()
+
+                for row in query:
+                    session = loads(dumps(dict(row), default=alchemyencoder))
```

Comment:
It returns a dictionary with `id` as `UUID(<value>)` it's not decoded. So either we will have to separately decode the value to str or use this JSON dumps.

Both dumps and loads are used because dumps return a string but we need it like a dictionary.